src/domain/enrolment/enrolmentForm/EnrolmentForm.tsx

```diff
@@ -115,15 +116,28 @@ const EnrolmentForm: React.FC<Props> = ({
   const { t } = useTranslation();
   const locale = useLocale();
 
-  const studyLevelOptions = Object.values(StudyLevel).map((level) => ({
-    label: level.startsWith('GRADE')
-      ? t('enrolment:studyLevel.grade_interval', {
-          postProcess: 'interval',
-          count: Number(level.split('_')[1]),
-        })
-      : translateValue('enrolment:studyLevel.', level, t),
-    value: level,
-  }));
+  const { data: studyLevelsData } = useStudyLevelsQuery();
+
+  const studyLevelOptions = studyLevelsData?.studyLevels?.edges?.map(
+    (studyLevelNodeEdge) => {
+      const level:
+        | string
+        | undefined = studyLevelNodeEdge?.node?.id.toUpperCase();
+
+      if (!level) {
+        return { label: '', value: '' };
+      }
+      return {
+        label: level.startsWith('GRADE')
+          ? t('enrolment:studyLevel.grade_interval', {
+              postProcess: 'interval',
+              count: Number(level.split('_')[1]),
+            })
+          : translateValue('enrolment:studyLevel.', level, t),
+        value: level,
+      };
+    }
+  );
```

Comment:
This whole section could be a hook in itself. That would be easier to reuse elsewhere.

Reply:
Created an `useStudyLevels` -hook in `domain/studyLevel`.